plugins/org.yakindu.sct.generator.c/src/org/yakindu/sct/generator/c/extensions/Naming.xtend

```diff
@@ -76,14 +76,6 @@ class Naming {
 		return entry.moduleName.toFirstUpper
 	}
 
-	def filterNullOrEmptyAndJoin(Iterable<CharSequence> it) {
-		filter[!it?.toString.nullOrEmpty].join('\n')
-	}
-
-	def filterNullOrEmptyAndJoin(List<String> it) {
-		filter[!it?.toString.nullOrEmpty].join('\n')
-	}
```

Comment:
The first function was used once, the second one never, so I inlined and deleted them.

Reply:
+1